server/utils/crmBackLink.ts

```diff
@@ -1,23 +1,15 @@
 import { Request } from 'express'
 
 function manageBackLink(req: Request, currentUrl: string): string {
```

Comment:
```suggestion
function manageBackLink(currentUrl: string): string {
```

Reply:
this needs to be left in otherwise it will trigger a currentUrl.includes is not a function error

Reply:
All calls like so `manageBackLink(req, currentUrl)` need to be changed to `manageBackLink(currentUrl)`

Reply:
done